src/main/java/net/sf/mzmine/modules/rawdatamethods/peakpicking/massdetection/Veritomyx/Veritomyx.java

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
Also, I think the package should change from ```massdetection.Veritomyx``` to ```massdetection.PeakInvestigator```, and this class should be ```PeakInvestigatorDetector```, to better match the naming conventions of other ```massdetection``` packages (e.g. https://github.com/mzmine/mzmine2/tree/master/src/main/java/net/sf/mzmine/modules/rawdatamethods/peakpicking/massdetection/localmaxima).